schedule/yast/encryption/crypt_no_lvm_pvm.yaml

```diff
@@ -15,34 +15,16 @@ vars:
   MAX_JOB_TIME: 14400
   YUI_REST_API: 1
 schedule:
-  - installation/bootloader_start
-  - installation/setup_libyui
-  - installation/product_selection/install_SLES
-  - installation/licensing/accept_license
-  - installation/registration/register_via_scc
-  - installation/module_registration/skip_module_registration
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_text_mode
-  - installation/partitioning/select_guided_setup
-  - installation/partitioning/guided_setup/encrypt_simple_pwd
-  - installation/partitioning/guided_setup/accept_default_fs_options
-  - installation/partitioning/accept_proposed_layout
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/use_same_password_for_root
-  - installation/authentication/default_user_simple_pwd
-  - installation/installation_settings/validate_ssh_service_enabled
-  - installation/installation_settings/open_ssh_port
-  - installation/bootloader_settings/disable_plymouth
-  - installation/bootloader_settings/disable_boot_menu_timeout
-  - installation/launch_installation
-  - installation/confirm_installation
-  - installation/performing_installation/perform_installation
-  - installation/logs_from_installation_system
-  - installation/performing_installation/confirm_reboot
-  - installation/handle_reboot
-  - installation/boot_encrypt
-  - installation/first_boot
-  - console/system_prepare
-  - console/validate_encrypt
+  guided_partitioning:
+    - installation/partitioning/select_guided_setup
+    - installation/partitioning/guided_setup/encrypt_simple_pwd
+    - installation/partitioning/guided_setup/accept_default_fs_options
+  first_login:
+    - installation/boot_encrypt
```

Comment:
this is actually ok because this phase is not login or or grub, but could you please put it under `grub` so we leave login appart.